src/Sylius/Bundle/ShopBundle/DependencyInjection/SyliusShopExtension.php

```diff
@@ -41,6 +41,7 @@ public function load(array $config, ContainerBuilder $container): void
         $loader->load(sprintf('services/integrations/locale/%s.xml', $config['locale_switcher']));
 
         $container->setParameter('sylius_shop.firewall_context_name', $config['firewall_context_name']);
+        $container->setParameter('sylius_shop.cart_session_key', $config['cart_session_key']);
```

Comment:
IMO instead of introducing this parameter we should use `CartStorage` instead (introduced in #8543) - the parameter itself does not separate concerns fully as there are still channel codes appended.